tools/hangar/__snapshots__/test_corpus/api_path_vars.w_test_sim.md

```diff
@@ -4,6 +4,6 @@
 ```log
 - Compiling to sim...
 ✔ Compiling to sim...
-pass ─ api_path_vars.wsim » root/env0/test
+pass ─ api_path_vars.wsim » root/env0/test:test
```

Comment:
can we special-case this to be "test"?

Reply:
I'll leave this as a future optimization